src/Package/Microsoft.Build.UnGAC/Microsoft.Build.UnGAC.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Setup runs on net45 and may not have installed newer yet. -->
+    <TargetFramework>net45</TargetFramework>
```

Comment:
Does that mean it doesn't work on mac/linux? Is there a way we can make it work on mac/linux?

Reply:
The GAC is a thing only on .NET Framework (on Windows). Things do not break because of the GAC on Mac/Linux, because there's no GAC.